### PR TITLE
Groceries

### PR DESCRIPTION
**Pull request by andreeahanson**
Hi, Chris!

I added some very important items to the grocery list. Please review and let me know what you think.

### CODE REVIEW

File: list.md [hunk 1]
```diff
@@ -2,3 +2,6 @@ eggs
 milk
 bread
 and whatever
+bread
```
Comment (chansonnier) [inline list.md]: I think we eat too much bread. Please reconsider this line. Thank you!

File: list.md [hunk 2]
```diff
@@ -2,3 +2,6 @@ eggs
...
+watermelon
```
Comment (chansonnier) [inline list.md]: What is watermelon?

File: list.md (hunk 1 above)
Comment (andreeahanson) [reply list.md]: Yes. I think so too. I'll change it to pizza instead.

File: list.md (hunk 2 above)
Comment (andreeahanson) [reply list.md]: It is water in the shape of a melon.

File: list.md [hunk 3]
```diff
@@ -2,6 +2,6 @@ eggs
 milk
 bread
 and whatever
-bread
+pizza
```
Comment (andreeahanson) [inline list.md]: See? I hope you're satisfied! Pizza is much better than bread. It has cheese! @chansonnier

Comment (chansonnier) [reply list.md]: Nice. Let's settle this once and for all. I am right, all the time. Thank you.

**Comment by chansonnier** [comment]
Good job! You're the best!

**Pull request merged**